addons/project/models/project.py

```diff
@@ -139,6 +139,12 @@ def write(self, vals):
             self.env['project.task'].search([('stage_id', 'in', self.ids)]).write({'active': False})
         return super(ProjectTaskType, self).write(vals)
 
+    def copy(self, default=None):
```

Comment:
Could you rephrase the commit like this?
```
[IMP] project: improve user experience

This commit introduces the following improvements:

- In `My Tasks` list view: the personal stage is displayed instead of the stage for private tasks
- When copying a stage, the new stage name is appended with `(copy)`
- A default filter 'open task' is addd in the subtask notebook search
- A default filter on the project is set on the search of the `parent_id` task field

task-3024152
```
The commit tiles should start with a verb.
The prior / after this commit is rather used when changing a business behavior and when it seems really important to remind on the previous behavior. I would say that here it was almost the repeating what has been introduced. That's why I would rather focus on what is introduced.

Reply:
Done, thanks for the precision about the commit message :)